jira_confluence_vpn_download_repair.ps1

```diff
@@ -1,9 +1,11 @@
 ﻿$USER = $env:UserName
+
+#Find the users's default browser and exectuable
 $BROWSER_PATH = "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"

```

Comment:
Is this necessary anymore? since the process is starting via line 8. The Browser Variable is no longer used.

Reply:
No, line 2 to line 4 can be removed safely